### DISCUSSION TITLE
New git folder and pages not appearing

### DISCUSSION THREAD

**Discussion by jlrouzies-fr** [requarks/wiki#5122] [Error / Bug Report]
Hello,

We are using WikiJS v2, and it is configured to take a GIT repo as data source, with a Postgress DB to store data.

![image](https://user-images.githubusercontent.com/57990274/160427767-e7414aad-13d3-4db7-95be-fc8ccc02d539.png)

Things are working fine, except recently we have 2 new folders with page inside, that just do not want to appear on the Wiki (we tried reboot, clear cache, etc):

1. Files are well downloaded locally

![image](https://user-images.githubusercontent.com/57990274/160428045-ad6028d8-cc72-44df-b6bc-2c69976e4f9d.png)

2. But on the Wiki, no trace of those folders:

![image](https://user-images.githubusercontent.com/57990274/160428135-a7d0472e-a762-4414-a014-374c9f938815.png)

What can we do about that?

Thanks

**Comment by jlrouzies-fr** [comment]
Any suggestions? It is really problematic to not be able to have new folders being taken into consideration.

**Comment by weber-s** [comment]
I recently ran into the same issue. For us, it appears that the folder didn't contains any "showable" file by wikijs (i.e. no `.md` in it).

Adding for instance a `Home.md` did fix the issue. Indeed wikijs does not handle file tree the traditional way (https://docs.requarks.io/guide/structure), and a `folder` in infer from page path, not filesystem tree.

However, it would be very nice if wikijs list "non-showable" file as well (assets (png, odt/docx, etc) in the file-tree !!

**Reply by jlrouzies-fr** [reply]
The issue is that we already have .md files inside

![image](https://user-images.githubusercontent.com/57990274/162150962-b707e8a8-329d-409c-a4a5-1a5b7461b47f.png)

**Comment by jlrouzies-fr** [comment]
Unfortunately the folders do contains wiki files (.md)
